docs/docs/configuration/live.md

```diff
@@ -0,0 +1,41 @@
+---
+id: live
+title: Live View
+---
+
+Frigate has different live view options, some of which require [restream](restream.md) to be enabled.
+
+## Live View Options
+
+Live view options can be selected while viewing the live stream. The options are:
+
+| Source | Latency | Frame Rate                             | Resolution     | Audio                       | Requires Restream | Other Limitations     |
+| ------ | ------- | -------------------------------------- | -------------- | --------------------------- | ----------------- | --------------------- |
+| jsmpeg | low     | same as `detect -> fps`, capped at 10  | same as detect | no                          | no                | none                  |
+| mp4    | high    | native                                 | native         | not yet                     | yes               | none                  |
+| webrtc | low     | native                                 | native         | yes (depends on browser)    | yes               | requires extra config |
+
+### webRTC extra configuration:
+
+webRTC works by creating a websocket connection on extra ports. One of the following is requried for webRTC to work:
```

Comment:
```suggestion
webRTC works by creating a websocket connection on extra ports. One of the following is required for webRTC to work:
```

I also think you should replace all `webRTC` with `WebRTC`, but that's up to you. :)